webapp/src/templates/partials/about.html

```diff
@@ -90,5 +90,14 @@ <h3 translate>debug.mode</h3>
       </form>
     </div>
 
+    <div ng-if="partners">
+      <h3 translate>partner.supporting</h3>
+      <ul class="list-inline">
+        <li ng-repeat="partner in partners">
+          <span ng-bind-html="partner.name | partnerImage"></span>
+        </li>
+      </ul>
+    </div>
```

Comment:
Is there CSS to constrain this image? I was expecting to see something like...

```css
img {
  max-height: 200px;
  max-width: 200px;
}
```

It'd also be worth trying to vertically centre the images if they're shorter than 100px and add margin to images that are thinner than 100px to help handle random sizes better. Also they'll need some margin between images horizontally and vertically.